### DISCUSSION TITLE
建议本地dns的udp dns走直连

### DISCUSSION THREAD

**Discussion by ssss123fff** [SagerNet/SagerNet#80] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
如果代理的线路不稳定，导致国内的dns解析很慢。
**Describe the solution you'd like**
本地dns的udp dns走直连
**Describe alternatives you've considered**
分应用，但是浏览器没法分应用
**Additional context**
![2021-05-09_11-17-45](https://user-images.githubusercontent.com/78684439/117559469-37b4d100-b0b8-11eb-9472-fcbe1dd0d76a.jpg)

**Comment by nekohasekai** [comment, marked as answer]
检查路由设置
